pandas/tests/tslibs/test_normalize_date.py

```diff
@@ -16,3 +17,22 @@
 def test_normalize_date(value, expected):
     result = tslibs.normalize_date(value)
     assert result == expected
+
+
+class SubDatetime(datetime):
+    pass
+
+
+@pytest.mark.parametrize("dt, expected", [
+    pytest.param(Timestamp(2000, 1, 1, 1),
+                 Timestamp(2000, 1, 1, 0)),
+    pytest.param(datetime(2000, 1, 1, 1),
+                 datetime(2000, 1, 1, 0)),
+    pytest.param(SubDatetime(2000, 1, 1, 1),
```

Comment:
i dont think the `pytest.param` is needed here

Reply:
Yeah, my bad.  I copied it from elsewhere and never removed it.